lib/Runtime/ByteCode/ByteCodeEmitter.cpp

```diff
@@ -2770,7 +2770,7 @@ void ByteCodeGenerator::EmitOneFunction(ParseNodeFnc *pnodeFnc)
 
     if (funcInfo->root->pnodeBody == nullptr)
     {
-        if (!PHASE_OFF1(Js::SkipNestedDeferredPhase))
```

Comment:
Can we no longer support creating/using stubs in a non-caching state? Seems like we might want to have that.

Reply:
Yes, we could support that but we'll need to do something to make it work. As it is, if we didn't capture names in the parse phase but we create stubs we will use the stubs to skip functions without pushing references to the captured names - since we don't collect them. That's why I added this check.